conf/eslint-recommended.js

```diff
@@ -212,7 +212,7 @@ module.exports = {
         "no-with": "off",
         "nonblock-statement-body-position": "off",
         "object-curly-newline": "off",
-        "object-curly-spacing": ["off", "never"],
+        "object-curly-spacing": "off",
```

Comment:
The default for `object-curly-spacing` is `"never"` anyway, so I don't think this was doing anything. It was added in https://github.com/eslint/eslint/commit/b2aedfe58c3569ff8e9caf132ad625cecac0308c and seems to have been a mistake.